creature.cpp

```diff
@@ -1144,6 +1144,8 @@ static string getAttackParam(AttackType type) {
     case AttackType::SPELL: return "spell";
     case AttackType::POSSESS: return "touch";
   }
+  assert("__FUNCTION__ case unknown attack type");
+  return "";
```

Comment:
Using clang version 3.5.0 (tags/RELEASE_350/final 216961, 3.5.0-7.1) with libboost 1.57

Gives similar error as https://github.com/miki151/keeperrl/issues/460

In file included from stdafx.h:152:
./serialization.h:288:6: fatal error: redefinition of 'serialize'
void serialize(Archive& ar, std::array<T,N>& a, const unsigned int)
     ^
/usr/include/boost/serialization/array.hpp:145:6: note: previous definition is here
void serialize(Archive& ar, std::array<T,N>& a, const unsigned int /* version */)
     ^